pkg/kwokctl/cmd/delete/cluster/cluster.go

```diff
@@ -73,41 +80,58 @@ func runE(ctx context.Context, flags *flagpole) error {
 	if err != nil {
 		if errors.Is(err, os.ErrNotExist) {
 			logger.Warn("Cluster does not exist")
+			if !force {
+				return nil
+			}
+		} else if !force {
+			return err
 		}
-		return err
 	}
 
```

Comment:
The default is now forced deletion, so you need to check the `rt.Available(ctx)` if not force and then quit.